docs/api/utils.md

```diff
@@ -244,75 +244,51 @@ const todoFamily = atomFamily(
 
 https://codesandbox.io/s/react-typescript-forked-8zfrn
 
-## useSelector
+## selectAtom
 
 Ref: https://github.com/pmndrs/jotai/issues/36
 
-### Usage
-
 ```js
-useSelector(anAtom, selector, equalityFn)
+function selectAtom<Value, Slice>(
+  anAtom: Atom<Value>,
+  selector: (v: Value) => Slice,
+  equalityFn: (a: Slice, b: Slice) => boolean = Object.is
+): Atom<Slice>
```

Comment:
Do you think it's better to have TS in docs? You want to change `ts` in L251 then.

(btw, we should work on improving docs, and maybe it's better to make it consistent throughout the docs. would be for another PR.)